doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                   |         merge bob--------&gt;|              |
+                   |                           |              |
+                   |                         commit           |
+                   |                           |-----update--&gt;|
+                   |                           |           commit
+                   |                           |&lt;-merge bob---|
+                   |                         commit
+                   |&lt;--merge branch------------|
+                   |
+                commit
+</pre>
+                Flow diagram of best working practice for local/working-copies and branches 
+                with branches of branches (bob).   
+            </div>
+            <div class="page-header">
+                <h3>Basics of using an existing Fcm repository</h3>
+            </div>
+            <div class="well">
+                <li>trunk v local</li>
+                "trunk" is the version which is committed to the repository and also known as 
+                the checked in version.
+                <br/>
+                "local" is also known as the working copy and it is where a copy of the 
+                checked out trunk was copied to for the user to develop.
+                <br/>
+                <li>
+                    Branching
+                </li>
+                Branching is cheap, therefore it is good practice to branch from the trunk 
+                of the repository.
+                Note we do not have to "recycle old branches" just create a new one.
+            </div>
+            <p>This tutorial is about a version control system developed at the Met Office, 
+                Flexible Configuration Management system, known as Fcm.
+            </p>
+            <div class="page-header">
+                <h3>What is Fcm?</h3>
+            </div>
+            <div class="well">
+            Fcm was designed to be a helpful and simple interface for users to version 
+            control work which is based on and built around Subversion, SVN.
+            <br/>
+            Fcm is an extension to SVN and adds features to make the user experience of 
+            managing their code and work easier.
+            <br/>
+            Fcm can be considered a wrapper to SVN.
+            <br/>
+            Fcm simplifies and enhances SVN to provide a user friendly interface to SVN 
+            therefore allowing users to effectively version control there work.
+            <br/>
+            Fcm links together SVN and TRAC, TRAC is an environment by which projects can 
+            be hosted and can be used to record information and tickets for issues can be 
+            hosted.
+            <br/>
+            <br/>
+            <h4>Fcm user guide</h4>
+            <ul>
+                <li><a href="http://www-nwp/~fcm/fcm-doc/fcm-2015.09.0/user_guide/getting_started.html">Getting started</a></li>
```

Comment:
Needs to be a relative link not an absolute one : ../user_guide/getting_started.html